04-onion-routing.md

```diff
@@ -254,14 +254,17 @@ The creator of `encrypted_recipient_data` (usually, the recipient of payment):
     - `total_fee_proportional_millionths(n+1) = ((total_fee_proportional_millionths(n) + fee_proportional_millionths(n+1)) * 1000000 + total_fee_proportional_millionths(n) * fee_proportional_millionths(n+1) + 1000000 - 1) / 1000000`
   - MUST create the `encrypted_recipient_data` from the `encrypted_data_tlv` as required in [Route Blinding](#route-blinding).
 
-The writer of `tlv_payload`:
+The writer of the TLV `payload`:
 
   - For every node inside a blinded route:
     - MUST include the `encrypted_recipient_data` provided by the recipient
     - For the first node in the blinded route:
       - MUST include the `blinding_point` provided by the recipient in `current_blinding_point`
     - If it is the final node:
       - MUST include `amt_to_forward`, `outgoing_cltv_value` and `total_amount_msat`.
+      - `outgoing_cltv_value` MUST be set to any additional CLTV delta which was added in addition
+        to the required amount for the full blinded path for privacy. This may be zero. This MUST
+        NOT include any additional amount added to compensate for a block during payment relay.
```

Comment:
Could we clarify that `outgoing_cltv_value` will still be absolute? It sounds like it's set to a delta right now